week-8/Homework/mandatory/3-dog-photo-gallery/script.js

```diff
@@ -0,0 +1,27 @@
+let happyButton = document.getElementById("getADog");
+let dogList = document.getElementById("dogList");
+
+happyButton.addEventListener("click", () => {
+  getData("https://dog.ceo/api/breeds/image/random");
+});
+function getData(data) {
+  fetch(data)
+    .then((response) => {
+      return response.json();
+    })
+    .then((dog) => {
+      dogList.append(makeListItem(dog));
+    })
+    .catch((error) => console.log(error));
```

Comment:
When you log an error, usually we write to the `error` log to indicate the type of the issue. The console has a bunch of features among others it can separate so-called log levels such as Log, Info, Debug, Warn and Error. Check them out and see the differences in the console.
https://www.youtube.com/watch?v=RLjhC5r1mRo

Reply:
This was very useful video. so line 15 should look like this :
`.catch((error) => console.error(error));`